CONTRIBUTING.md

```diff
@@ -45,7 +45,7 @@ npm run preview
 				"PRISMIC_ENV": "staging",
 
 				# Activate debug mode to see more logs (Output tab of your IDE console)
-				"DEBUG": true
+				"PRISMIC_DEBUG": true
```

Comment:
Renamed as I want to ensure it doesn't mix with other env variables, for example, the one for playwright `DEBUG`